spec/support/vcr_setup.rb

```diff
@@ -6,6 +6,11 @@
   c.hook_into :webmock
   c.allow_http_connections_when_no_cassette = true
   c.configure_rspec_metadata!
+  c.filter_sensitive_data('UA-BOGUS') { ENV['GOOGLE_ANALYTICS_ID'] }
+  c.filter_sensitive_data('BOGUS APP') { ENV['GOOGLE_ANALYTICS_APP_NAME'] }
+  c.filter_sensitive_data('BOGUS/PATH') { ENV['GOOGLE_ANALYTICS_PRIVKEY_PATH'] }
+  c.filter_sensitive_data('BOGUS SECRET') { ENV['GOOGLE_ANALYTICS_PRIVKEY_SECRET'] }
+  c.filter_sensitive_data('EMAIL@BOGUS.COM') { ENV['GOOGLE_ANALYTICS_CLIENT_EMAIL'] }
```

Comment:
...why do we need the VCR setup? :sheep:

It seems like patching ('doubling' in RSpec-speak I believe) `FileUsage` would be much easier and faster (for the tests to run too). Try it out by checking this for instance: https://semaphoreci.com/community/tutorials/mocking-with-rspec-doubles-and-expectations

Reply:
When the `FileUsage` object initializes, it makes a request to get the stats from Google analytics, hence the VCR use here. Maybe I could use doubling for testing here. I will take a look and follow up if I will implement it instead.